js/ui/number_box/number_box.mask.js

```diff
@@ -12,6 +12,7 @@ const getLDMLFormat = require('../../localization/ldml/number').getFormat;
 const NumberBoxBase = require('./number_box.base');
 const eventUtils = require('../../events/utils');
 const typeUtils = require('../../core/utils/type');
+const { ensureDefined } = require('../../core/utils/common');
```

Comment:
```suggestion
const { ensureDefined, escapeRegExp } = require('../../core/utils/common');
```
and remove [R8](https://github.com/DevExpress/DevExtreme/pull/12653/commits/60c2c8e3dcbf23d349ee6461fd3ee8cba6b4a01f#diff-e72e966671af97ad50c72391349a011bR8)